docs/apache-airflow-providers-amazon/logging/s3-task-handler.rst

```diff
@@ -37,13 +37,16 @@ To enable this feature, ``airflow.cfg`` must be configured as follows:
     # id that provides access to the storage location.
     remote_logging = True
     remote_base_log_folder = s3://my-bucket/path/to/logs
-    remote_log_conn_id = MyS3Conn
+    remote_log_conn_id = aws_s3_conn
     # Use server-side encryption for logs stored in S3
     encrypt_s3_logs = False
 
-In the above example, Airflow will try to use ``S3Hook('MyS3Conn')``.
+In the above example, Airflow will try to use ``S3Hook(aws_conn_id='aws_s3_conn')``.
+
+Local test s3 remote logging
+''''''''''''''''''''''''''''
 
 You can also use `LocalStack <https://localstack.cloud/>`_ to emulate Amazon S3 locally.
 To configure it, you must additionally set the endpoint url to point to your local stack.
-You can do this via the Connection Extra ``host`` field.
-For example, ``{"host": "http://localstack:4572"}``
+You can do this via the Connection Extra ``endpoint_url`` field.
+For example, ``{"endpoint_url": "http://localstack:4572"}``
```

Comment:
what does this change have to do with deprecating the conn type?

Reply:
Just noticed that in the documentation page contain outdated info and decide to change it in one go rather than separate PR. Ig it is a problem I could revert it back.